app/screens/bottom_sheet/index.tsx

```diff
@@ -19,24 +19,27 @@ import Indicator from './indicator';
 
 type SlideUpPanelProps = {
     closeButtonId?: string;
+    componentId?: string;
     initialSnapIndex?: number;
     renderContent: () => ReactNode;
     snapPoints?: Array<string | number>;
 }
 
-const BottomSheet = ({closeButtonId, initialSnapIndex = 0, renderContent, snapPoints = ['90%', '50%', 50]}: SlideUpPanelProps) => {
+const BottomSheet = ({closeButtonId, componentId: screenComponentId, initialSnapIndex = 0, renderContent, snapPoints = ['90%', '50%', 50]}: SlideUpPanelProps) => {
```

Comment:
why do you need the componentId here? this is fine, just want to understand the use case as I fail to see it in this code

Reply:
In tablet, we are not using the `bottomsheet` but in the `close` function, we are dismissing the `bottomsheet` because of which `PostOptions` is not closing on tablets and throwing an error: "`bottomsheet` is not a modal"

So, for the tablets, we pop the modal screen instead of this. This change is only for the bottomsheet to work in tablets.    If changing the prop name or adding a comment might help us understand the change better, I will add it.

Reply:
Hmmmm, definitely not sure I follow what you are saying, how is not a modal? And if it is not, then it should. Post_options on the other hand is it's own modal that uses the bottom sheet component

Reply:
For the tablet, we are not rendering `BottomSheet` so if we dismiss it, it would throw an error. That's why I am passing the componentId and for the tablet, we will dismiss the `componentId`. 

https://github.com/mattermost/mattermost-mobile/blob/1e1992e979f8d3fa0fe84f7b13041e4b37cf47c2/app/screens/bottom_sheet/index.tsx#L109-L135

Reply:
Added componentId prop and `close` function as discussed.